portal/server/configuration-loader.ts

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+type StringBoolean = 'true' | 'false';
+function isStringBoolean(value: string): value is StringBoolean {
+    return value === 'true' || value === 'false';
+}
+function toBoolean(value: string): Boolean {
+    return value === 'true';
+}
```

Comment:
Question: I guess there is no way to store a private yaml file or similar instead of env variables?

Reply:
Yea there is no such built-in functionality in Vercel. Other custom options (such as using vercel's blob store to save yaml files, and then load them on the portal) seemed to overcomplicate the logic.